src/components/ADempiere/Form/VPOS/Collection/typeCollection.vue

```diff
@@ -69,16 +69,31 @@
                       >
                         <p class="total">
                           <b v-if="!isEmptyValue(value.multiplyRate)" style="float: right;">
-                            {{ formatPrice(value.multiplyRate, currency.iSOCode) }}
+                            {{
+                              formatPrice({
+                                value: value.multiplyRate,
+                                currencyCode: currency.iSOCode
+                              })
+                            }}
                           </b>
                           <b v-else style="float: right;">
-                            {{ formatPrice(value.amount, currency.iSOCode) }}
+                            {{
```

Comment:
Just can see change for source code with the same functionality, please let it without changes